Building TX16s firmware with -DDEBUG=YES -DCMAKE_BUILD_TYPE=Debug fails

### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

Building TX16s firmware with -DDEBUG=YES -DCMAKE_BUILD_TYPE=Debug fails with error message
```
[ 38%] Building CXX object radio/src/targets/common/arm/stm32/bootloader/CMakeFiles/bootloader.dir/__/__/__/__/__/serial.cpp.obj
In file included from C:/xxx/EdgeTX/radio/src/tasks/mixer_task.h:22,
                 from C:/xxx/EdgeTX/radio/src/serial.cpp:34:
C:/xxx/EdgeTX/radio/src/rtos.h:318:4: error: #error "No RTOS implementation defined"
  318 |   #error "No RTOS implementation defined"
      |    ^~~~~
```

### Expected Behavior

Don't fail

### Steps To Reproduce

Build TX16s firmware with -DDEBUG=YES -DCMAKE_BUILD_TYPE=Debug

### Version

Nightly (Please give date/commit below)

### Transmitter

Radiomaster TX16S / TX16SMK2

### Operating System (OS)

_No response_

### OS Version

_No response_

### Anything else?

_No response_

--- a/radio/src/serial.cpp
+++ b/radio/src/serial.cpp
@@ -29,16 +29,16 @@
 
 #include "hal/serial_port.h"
 
+#if defined(CONFIGURABLE_MODULE_PORT) and !defined(BOOT)
+  #include "hal/module_port.h"
+  #include "tasks/mixer_task.h"
+#endif
+
 #if !defined(BOOT)
   #include "opentx.h"
   #include "lua/lua_api.h"
 #else
   #include "dataconstants.h"
-#endif
-
-#if defined(CONFIGURABLE_MODULE_PORT)
-  #include "hal/module_port.h"
-  #include "tasks/mixer_task.h"
 #endif
 
 #if defined(CROSSFIRE)
@@ -256,7 +256,7 @@ static void serialSetCallBacks(int mode, void* ctx, const etx_serial_port_t* por
     break;
 #endif
 
-#if defined(CONFIGURABLE_MODULE_PORT)
+#if defined(CONFIGURABLE_MODULE_PORT) and !defined(BOOT)
   case UART_MODE_EXT_MODULE:
     if (port && !ctx) { // de-init
       etx_module_port_t mod_port;
@@ -400,7 +400,7 @@ void serialInit(uint8_t port_nr, int mode)
     memset(state, 0, sizeof(SerialPortState));
   }
 
-#if defined(CONFIGURABLE_MODULE_PORT)
+#if defined(CONFIGURABLE_MODULE_PORT) and !defined(BOOT)
   if (mode == UART_MODE_EXT_MODULE) {
     etx_module_port_t mod_port = {
       .port = ETX_MOD_PORT_UART,

--- a/radio/src/serial.cpp
+++ b/radio/src/serial.cpp
@@ -29,16 +29,15 @@
 
 #include "hal/serial_port.h"
 
-#if defined(CONFIGURABLE_MODULE_PORT)
-  #include "hal/module_port.h"
-  #include "tasks/mixer_task.h"
-#endif
-
 #if !defined(BOOT)
   #include "opentx.h"
   #include "lua/lua_api.h"
 #else
   #include "dataconstants.h"
+#endif
+
+#if defined(CONFIGURABLE_MODULE_PORT)
+  #include "hal/module_port.h"
 #endif
 
 #if defined(CROSSFIRE)

--- a/radio/src/serial.cpp
+++ b/radio/src/serial.cpp
@@ -38,6 +38,7 @@
 
 #if defined(CONFIGURABLE_MODULE_PORT)
   #include "hal/module_port.h"
+  #include "tasks/mixer_task.h"
 #endif
 
 #if defined(CROSSFIRE)

